testing/acceptance_test.py

```diff
@@ -1390,3 +1392,63 @@ def test_boo(self):
     )
     result = pytester.runpytest_subprocess()
     result.stdout.fnmatch_lines("*1 passed*")
+
+
+def test_issue_9765(pytester: Pytester) -> None:
```

Comment:
```suggestion
@pytest.mark.skip(reason="Test is not isolated")
def test_issue_9765(pytester: Pytester) -> None:
```

Reply:
I skipped the test for now, but I may consider doing a follow-up PR to add venv creation and activation in this test, via `python -m venv` + `activate`, would you be OK with this?

Reply:
venv creation is quite heavy and slow, and the setuptools thing still remains, so I'd still prefer it skipped